.github/workflows/checkstyle.yml

```
@@ -0,0 +1,26 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Run Checkstyle
+        uses: nikitasavinov/checkstyle-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          reporter: 'github-pr-check'
+          tool_name: 'checkstyle'
+          level: 'error'
+          fail_on_error: true
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)